pkg/install/v1alpha1/jiva_snapshot.go

```diff
@@ -31,18 +31,34 @@ spec:
   taskNamespace: {{env "OPENEBS_NAMESPACE"}}
   run:
     tasks:
-    - jiva-volume-podsinopenebsns-default
+    - jiva-snapshot-podsinopenebsns-default
     - jiva-snapshot-create-listsourcetargetservice-default
     - jiva-snapshot-create-invokehttp-default
   output: jiva-snapshot-create-output-default
 ---
 apiVersion: openebs.io/v1alpha1
 kind: RunTask
+metadata:
+  name: jiva-snapshot-podsinopenebsns-default
+spec:
+  meta: |
+    id: jivasnappodsinopenebsns
+    runNamespace: {{ .Config.OpenEBSNamespace.value }}
+    apiVersion: v1
+    kind: Service
+    action: list
+    options: |-
+      labelSelector: openebs.io/controller-service=jiva-controller-svc,openebs.io/persistent-volume={{ .Snapshot.volumeName }}
+  post: |
+    {{- jsonpath .JsonResult "{.items[*].metadata.namespace}" | trim | saveAs "jivasnappodsinopenebsns.ns" .TaskResult | noop -}}
```

Comment:
listing a service require namespace also, by default it will search in default namespace with the given labels. @AmitKumarDas how runtask process this post request ? does this list in runNamespace ?

Reply:
yes @utkarshmani1997 .. it uses the runNamespace.

Reply:
for exp:
![image](https://user-images.githubusercontent.com/19219723/57617918-931e5800-759f-11e9-9004-782029c3fa07.png)

Reply:
@kmova so, if user will deploy in "default" namespace, what will be value of "runNamespace", will it be overridden to "default" ?